### PR TITLE
fix vue tour problems

### PR DESCRIPTION
**Pull request by Amirreza-Ghasemkhani**
https://github.com/project-chip/zap/pull/625

### CODE REVIEW

File: src/components/ZclEndpointCard.vue [hunk 1]
```diff
@@ -317,7 +293,7 @@ export default {
               }
             }
           })
-          this.selectedservers = [...enabledServers, ...enabledServers]
+          this.selectedservers = [...enabledServers, ...enabledClients]
```
Comment (jingteng25742) [inline src/components/ZclEndpointCard.vue:296]: why did it change to enabledClients?

File: src-electron/db/query-package.js [hunk 2]
```diff
@@ -71,6 +71,25 @@ async function getPackageByParent(db, parentId) {
     .then((rows) => rows.map(dbMapping.map.package))
 }
 
+/**
```
Comment (jingteng25742) [inline src-electron/db/query-package.js:74]: @Amirreza-Ghasemkhani , 
I don't think this is part of our vue tour code. 
Did this get accidentally introduced as part of a rebase/merge?

File: src-electron/generator/matter/app/zap-templates/common/ChipTypesHelper.js [hunk 3]
```diff
@@ -54,5 +54,5 @@ function asBasicType(type) {
 exports.asBasicType = asBasicType;
 exports.meta = {
   category: dbEnum.helperCategory.matter,
-  alias: ['common/ChipTypesHelper.js'],
+  alias: ['common/ChipTypesHelper.js', 'matter-chip-types-helper'],
```
Comment (jingteng25742) [inline src-electron/generator/matter/app/zap-templates/common/ChipTypesHelper.js:57]: accident w/ merge?

**Review by jingteng25742 — CHANGES_REQUESTED** [review]
A lot of matter code seem to be accidentally introduced as part of the merge.

File: src/components/ZclEndpointCard.vue (hunk 1 above)
Comment (Amirreza-Ghasemkhani) [reply src/components/ZclEndpointCard.vue:296]: Here we have to enableCluster but we should use enableClient for one of them I change that to fix issue

File: src-electron/db/query-package.js (hunk 2 above)
Comment (Amirreza-Ghasemkhani) [reply src-electron/db/query-package.js:74]: I tried to fix vuetour on matter and for that I need allpackages or uioption I tried to fix that but we need work on some of apies so after discussion we decide to work on this problem in another branch

File: src-electron/generator/matter/app/zap-templates/common/ChipTypesHelper.js (hunk 3 above)
Comment (Amirreza-Ghasemkhani) [reply src-electron/generator/matter/app/zap-templates/common/ChipTypesHelper.js:57]: After rebasing that with master this was appeared I think

**Pull request closed without merging**